core-build/gulp-core-build-karma/tsconfig.json

```diff
@@ -1,13 +1,17 @@
 {
   "compilerOptions": {
-    "target": "es5",
+    "target": "es6",
     "module": "commonjs",
     "jsx": "react",
     "declaration": true,
     "sourceMap": true,
     "strictNullChecks": true,
     "types": [
       "node"
+    ],
+    "lib": [
+      "es5",
+      "es2015.iterable"
```

Comment:
Does this package need those libraries?

Reply:
Yeah, I was surprised, but I needed es2015.iterable in a lot more places than I thought.. If I remove it you get an error like this:

```
 typescript - unknown filename(0,0): error TS2318: Cannot find global type 'Symbol'.
```